NUMERO_ETUDIANT    27122013146932264

CIVILITE         Mme
NOM              SOL
PRENOM           ROSE
DATE_NAISSANCE   2006_05_15
TELEPHONE_FIXE   0354674581
TELEPHONE_MOBILE 0672216337
EMAIL            sol.rose4717@laposte.net

VOIE             131 RUE DES MERLES
CODE_POSTAL      57420
COMMUNE          POUILLY

04/11/2024 - 3 - SyntaxError - invalid syntax (detected at line 41) - enlever la virgule
25/09/2024 - 2 - AttributeError - 'str' object has no attribute 'keys' - convertir en dictionnaire ou vérifier l'objet
14/10/2024 - 1 - SyntaxError - too many positional arguments - vérifier les arguments d'une fonction
18/11/2024 - 1 - NameError - name 'main' is not defined. Did you mean '__main__'? - corriger en __main__
02/10/2024 - 1 - TypeError - unsupported operand type(s) for +: 'set' and 'set' - remplacer + par union
02/10/2024 - 3 - NameError - name 'fuction' is not defined. Did you mean 'function'? - corriger en function
18/11/2024 - 2 - IndexError - index out of range in self - corriger l'indice pour qu'il soit dans les limites de l'objet
08/11/2024 - 1 - IndentationError - expected an indented block after 'case' statement on line 19 - ajouter un bloc indenté après case
25/11/2024 - 1 - SyntaxError - unmatched ')' - enlever la parenthèse en trop
14/10/2024 - 1 - IndexError - array index out of range - assurez-vous que l'indice est dans les limites du tableau
04/11/2024 - 2 - AttributeError - 'list' object has no attribute 'update' - remplacer update par append ou extend pour les listes
14/10/2024 - 5 - SyntaxError - unexpected token '=' - vérifier les affectations
12/11/2024 - 1 - NameError - name 'lengh' is not defined. Did you mean 'length'? - corriger en length
25/11/2024 - 2 - NameError - name 'indet' is not defined. Did you mean 'indent'? - corriger en indent
08/11/2024 - 2 - IndentationError - unexpected unindent in 'with' statement block - corriger l'indentation dans le bloc with
25/11/2024 - 3 - AttributeError - 'str' object has no attribute 'keys' - convertir en dictionnaire ou vérifier l'objet
18/11/2024 - 1 - TypeError - 'dict' object is not callable - vérifier si un dictionnaire est utilisé comme une fonction
12/11/2024 - 1 - TypeError - 'resumer' is an invalid keyword argument for print() - définir r
25/09/2024 - 4 - SyntaxError - unexpected 'await' outside async function - placer 'await' dans une fonction async
14/10/2024 - 2 - SyntaxError - missing 'as' in 'with' statement - ajouter 'as' dans le bloc with
02/12/2024 - 1 - NameError - name 'filtre' is not defined. Did you mean 'filter'? - corriger en filter
25/11/2024 - 1 - IndentationError - expected an indented block after 'case' statement on line 19 - ajouter un bloc indenté après case
02/12/2024 - 2 - SyntaxError - unexpected non-keyword after *args - vérifier la position des *args dans les arguments
4/09/2024 - 4 - NameError - name 'main' is not defined. Did you mean '__main__'? - corriger en __main__